README.rst

```diff
@@ -97,9 +97,13 @@ Running tests
 
 SageMaker Python SDK has unit tests and integration tests.
 
+You can install the libraries needed to run the tests by running :code:`pip install --upgrade .[test]` or, for Zsh users: :code:`pip install --upgrade .\[test\]`
+
 **Unit tests**
 
-tox is a prerequisite for running unit tests so you need to make sure you have it installed. To run the unit tests:
+To run the unit tests, you'll need to have interpreters for Python 2.7 and Python 3.5 installed.
```

Comment:
this isn't strictly true. one can have only one version of Python installed and run `pytest test/unit`

Reply:
The tox.ini says it will run with Python 2.7 and Python 3.5 interpreters -- I only have python 3.6 installed, and tox complained about that: ERROR:   py35: InterpreterNotFound: python3.5

Reply:
right. my point is that you technically don't have to use tox to run the unit tests at all (the previous README sentence was incorrect) - I think I'd change this part to convey "we test with Python 2.7 and 3.5, you can use this tool called tox to run the same unit test setup we do, and you'll need those interpreters installed"

Reply:
OK, I changed it